content/en/docs/userguide/install_waypoint.md

```diff
@@ -76,19 +78,9 @@ Label the `reviews` service to use the `reviews-svc-waypoint` waypoint:
 
 ```bash
 [root@ ~]# kubectl label service reviews istio.io/use-waypoint=reviews-svc-waypoint
-[root@ ~]# kubectl get pods
-NAME                                      READY   STATUS    RESTARTS   AGE
-details-v1-cdd874bc9-xcdnj                1/1     Running   0          30m
-productpage-v1-5bb9985d4d-z8cws           1/1     Running   0          30m
-ratings-v1-6484d64bbc-pkv6h               1/1     Running   0          30m
-reviews-svc-waypoint-8cb4bdbf-9d5mj       1/1     Running   0          30m
-reviews-v1-598f9b58fc-2rw7r               1/1     Running   0          30m
-reviews-v2-5979c6fc9c-72bst               1/1     Running   0          30m
-reviews-v3-7bbb5b9cf7-952d8               1/1     Running   0          30m
-sleep-5577c64d7c-n7rxp                    1/1     Running   0          30m
 ```
-  
-Replace the waypoint image with the Kmesh customized image. Based on istio-proxy, Kmesh adds an customized listener filter called [Kmesh_tlv](https://github.com/kmesh-net/waypoint/tree/master/source/extensions/filters/listener/kmesh_tlv), which will parse the custom TLV protocol encoded by Kmesh and obtain the target address and metadata to connect L4 and L7.
+
+You can use`kubectl get pods` to see all the pods except waypoint are ready. Then replace the waypoint image with the Kmesh customized image. Based on istio-proxy, Kmesh adds an customized listener filter called [Kmesh_tlv](https://github.com/kmesh-net/waypoint/tree/master/source/extensions/filters/listener/kmesh_tlv), which will parse the custom TLV protocol encoded by Kmesh and obtain the target address and metadata to connect L4 and L7.
```

Comment:
As discussed in https://github.com/kmesh-net/kmesh/issues/678, highlight image replacement operation to remind user.